build_tools/packaging/linux/package.json

```
@@ -207,103 +256,2443 @@
+      "rccl"
+    ],
+    "Artifact": "rccl",
+    "Gfxarch": "True",
+    "Composite": "Yes"
+  },
+
+  {
+    "Package": "rocm-rccl-devel",
+    "Version": "",
+    "Architecture": "amd64",
+    "BuildArch": "x86_64",
+    "DEBDepends": [
+      "python3",
+      "libc6",
+      "rocm-rccl"
+    ],
+    "RPMRequires": [
+      "python3",
+      "glibc",
+      "rocm-rccl"
+    ],
+    "Maintainer": "RCCL Maintainer <rccl-maintainer@amd.com>",
+    "Description_Short": "ROCm Communication Collectives Library. Optimized primitives for collective multi-GPU communication",
+    "Description_Long": "ROCm Communication Collectives Library. Optimized primitives for collective multi-GPU communication",
+    "Section": "devel",
+    "Priority": "optional",
+    "Group": "unknown",
+    "License": "MIT",
+    "Vendor": "Advanced Micro Devices, Inc",
+    "Homepage": "https://github.com/ROCm/rccl",
+    "Components": [
+      "dev"
+    ],
+    "Includes": [
+      "rccl-devel"
+    ],
+    "Artifact": "rccl",
+    "Gfxarch": "True",
+    "Composite": "Yes"
+  },
+  {
+    "Package": "rocm-miopen",
+    "Version": "",
+    "Architecture": "amd64",
+    "BuildArch": "x86_64",
+    "DEBDepends": [],
+    "RPMRequires": [],
+    "Maintainer": "MIOpen Maintainer <miopen-lib.support@amd.com>",
+    "Description_Short": "AMD DNN Library",
+    "Description_Long": "MIOpen is AMD's library for high-performance machine learning primitives",
+    "Section": "devel",
+    "Priority": "optional",
+    "Group": "unknown",
+    "License": "MIT",
+    "Vendor": "Advanced Micro Devices, Inc",
+    "Homepage": "https://github.com/ROCm/rocm-libraries",
+    "Components": [
+      "lib",
+      "run",
+      "doc"
+    ],
+    "Artifact": "miopen",
+    "Gfxarch": "True",
+    "Composite": "Yes"
+  },
+
+  {
+    "Package": "rocm-miopen-devel",
+    "Version": "",
+    "Architecture": "amd64",
+    "BuildArch": "x86_64",
+    "DEBDepends": [
+      "rocm-miopen"
+    ],
+    "RPMRequires": [
+      "rocm-miopen"
+    ],
+    "Maintainer": "MIOpen Maintainer <miopen-lib.support@amd.com>",
+    "Description_Short": "AMD DNN Library",
+    "Description_Long": "Provides development files for miopen library",
+    "Section": "devel",
+    "Priority": "optional",
+    "Group": "unknown",
+    "License": "MIT",
+    "Vendor": "Advanced Micro Devices, Inc",
+    "Homepage": "https://github.com/ROCm/rocm-libraries",
+    "Components": [
+      "dev"
+    ],
+    "Artifact": "miopen",
+    "Gfxarch": "True",
+    "Composite": "Yes"
+  },
+
+  {
+    "Package": "rocm-prim",
+    "Version": "",
+    "Architecture": "amd64",
+    "BuildArch": "x86_64",
+    "DEBDepends": [
+      "python3",
+      "libc6"
+    ],
+    "RPMRequires": [
+      "python3",
+      "glibc"
+    ],
+    "Maintainer": "ROCm Dev Support <rocm-dev.support@amd.com>",
+    "Description_Short": "ROCM primitive header files",
+    "Description_Long": "Contains rocprim, hipcub and rocthrust developement files",
```

fix capitalization for ROCm? here and in a few other places
```suggestion
    "Maintainer": "ROCm Dev Support <rocm-dev.support@amd.com>",
    "Description_Short": "ROCm primitive header files",
    "Description_Long": "Contains rocprim, hipcub and rocthrust developement files",
```

---

ROCm and LLVM capitalized in all descriptions
FYI,  The description  and dependencies  defined for composite(group) packages are not finalized.  Will be updating as we get more details on these packages